### ISSUE TITLE
mysql_connect deprecated

### ISSUE THREAD

**Issue by jUsjDnr9vNok4kelSGyl** [jethrocarr/namedmanager#28]
Is someone working on refactoring the mysql_* functions to use MySQLi or DPO functions? Now that Ubuntu 16.04 LTS ships with PHP 7.0, the mysql_ functions are no longer available?

**Comment by jethrocarr** [comment]
Yeah these need to be converted to mysqli. If someone wants to run https://github.com/philip/MySQLConverterTool and do the required testing, I'll be happy to review and merge a PR for this.
